bot/exts/fun/quack.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+import discord
+from aiohttp import client_exceptions
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quack(commands.Cog):
+    """Cog for the .quack command."""
```

Comment:
Perhaps we could make it more ambiguous (name of the class included) and say that it's for wrapping Quackstack.